Formula/ceres-solver.rb

```diff
@@ -30,12 +29,6 @@ class CeresSolver < Formula
   depends_on "suite-sparse"
   depends_on "tbb"
 
-  on_linux do
-    depends_on "gcc"
-  end
-
-  fails_with gcc: "5" # C++17
```

Comment:
We should keep this line. The rest can be removed.

Reply:
I'm not a linux brew user, but I don't understand what that line would achieve on its own? (And it probably also fails with some other, later gcc versions.)

Reply:
I think it should help prevent `brew` from trying to use a host `gcc-5` if that's all that's available, and we already know that that's hopeless.